README.md

```diff
@@ -48,15 +58,51 @@ setups. Let us know if this is interesting for you.
 
 ## Configuration of the generated YAML file
 
-The setup command generates also a YAML file (default filename: `docker-compose.yml`)
-with the container configuration for all services. This step can be configured with
-a YAML formated config file. E. g. to get a customized YAML file run:
+The setup command generates also a YAML file (default filename:
+`docker-compose.yml`) with the container configuration for all services. This
+step can be configured with a YAML formated config file. E. g. to get a
+customized YAML file run:
 
     $ ./openslides setup --config my-config.yml .
 
+To rebuild the YAML file without resetting the whole directory (including
+secrets) run:
+
+    $ ./openslides config --config my-config.yml .
+
 See the [default config](pkg/setup/default-config.yml) for syntax and defaults.
 
 
+## HTTPS
+
+The manage tool provides two settable options for using HTTPS, both of which can
+be set in a `config.yml` file. When running OpenSlides on localhost a locally
+generated self-signed certificate can be used. To do so add the following line
```

Comment:
I don't think the certs have to be self signed. I think it is possible to use any cert you have with this method.

Reply:
You're actually right about that, I'll adjust the text